### DISCUSSION TITLE
Python: 'C:\Users\xxx' is not recognized as an internal or external command, operable program or batch file.

### DISCUSSION THREAD

**Discussion by uwatude** [community/community#37719] [Other Feature Feedback, Questions, & Ideas] [answered]
Sorry It might a silly question but...
I'm trying to run the following codes on Jupyter notebook.
---
import speedtest

test = speedtest.Speedtest()

down_speed = test.download()
up_speed = test.upload()
----
→　Then got this error:   　ModuleNotFoundError: No module named 'speedtest'

Then I realized I have to install "speedtest", so I ran this command:
---
pip install speedtest
---
→ Then I got this one:
------------
Note: you may need to restart the kernel to use updated packages.

'C:\Users\xxx' is not recognized as an internal or external command,
operable program or batch file.
------------
At this point, I'm not sure how to fix this error.
(I've restarted the kernel several times but no luck)

Could you please advise me how to fix this.
Thank you in advance!!!

**Comment by nalsur-veallam** [comment, marked as answer]
To fix this, you first need to uninstall the old speedtest using the following command:
> pip uninstall speedtest

And then install by running the following:
> pip install speedtest-cli

Don't forget to restart the kernel! (after that everything should work)

**Reply by uwatude** [reply]
I see Thank you!!!
